src/base64Encoder.ts

```diff
@@ -10,17 +11,17 @@ export function wrap(text: string, wrapAt: number): string {
     return result;
 }
 
-export function base64Encode(text: string, encodingName: string): string {
+export function base64Encode(text: string, encoding: string): string {
     try {
-        const data: Buffer = encode(text, encodingName);
-        const decodedString = decode(data, encodingName);
+        const data: Buffer = encode(text, encoding);
+        const decodedString = decode(data, encoding);
         if (text != decodedString) {
-            return `Input contains characters outside of ${encodingName}`;
+            return `Input contains characters outside of ${encoding2name.get(encoding)}`;
         }
         const binaryString = data.toString("binary");
         return btoa(binaryString);
     } catch (e) {
-        return `Input contains characters outside of ${encodingName}`;
+        return `Input contains characters outside of ${encoding}`;
```

Comment:
why use encoding in this error message and encoding2name.get(...) in the other one?

Reply:
Fixed.
It's not the greatest solution anyway. This was caused by two different values for the same encoding. For instance, I had to use both _UTF-8 and UTF-8 as encoding values, so that it can appear once under "Popular" and once under "Unicode" group. Now I have to deal with that hack here.
I don't have an idea that is better though. You could create a fully blown "dropdown" ui element that takes not only string values, but objects of an interface that controls a value of that object type, but imo that's too much code for too little gain.
Maybe in the future when we might want a dropdown that can also auto-complete what the user enters.